app/main.py

```diff
@@ -1,3 +1,59 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> None:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> None:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: float) -> None:
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        else:
+            return Distance(self.km + other)
```

Comment:
Else is redundant here
```suggestion
        return Distance(self.km + other)
```